src/drivers/planetscale.ts

```diff
@@ -0,0 +1,86 @@
+import { defineDriver } from "./utils";
+import type { ExecutedQuery } from "@planetscale/database";
+import { connect } from "@planetscale/database";
+import { fetch } from "ofetch";
+
+export interface PlanetscaleDriverOptions {
+  url?: string;
+  table?: string;
+  cache?: boolean;
+}
+
+interface TableSchema {
+  id: string;
+  value: string;
+  created_at: Date;
+  updated_at: Date;
+}
+
+export default defineDriver((opts: PlanetscaleDriverOptions = {}) => {
+  if (!opts.url)
+    throw new Error("Database URL is required to use the Planetscale driver.");
+
+  opts.table = opts.table || "storage";
+
+  // `connect` configures a connection class rather than initiating a connection
+  const c = connect({
+    url: opts.url,
+    fetch,
+  });
+
+  // this query will be lazily executed
+  if (opts.cache) {
+    c.execute("SET @@boost_cached_queries = true;");
```

Comment:
Are you sure it is lazy? Both return type (c.execute returns promise) and implementation i can find in `@planetscale/database` are returning promise:

<img width="697" alt="image" src="https://user-images.githubusercontent.com/5158436/216975476-8654f75f-8f8c-4437-9cec-f8383c1f3bdb.png">

Reply:
Yes, but we don't await it.

Reply:
It is not lazy executed only background this way... And will cause the server to make a post request on server startup/initializing drivers even if we don't need it immediately.

Reply:
What would you prefer? We can drop the feature or wrap this execution?

Reply:
Can you please verify last changes?

Reply:
looks perfect to me 👌